api/mock/facade.go

```diff
@@ -28,6 +29,15 @@ type Facade struct {
 	GetTransactionByHashAndSenderAddressHandler func(txHash string, sndAddr string) (*transaction.ApiTransactionResult, int, error)
 }
 
+// IsFaucetEnabled -
+func (f *Facade) IsFaucetEnabled() bool {
+	if f.IsFaucetEnabledHandler != nil {
+		return f.IsFaucetEnabledHandler()
+	}
+
+	return true
```

Comment:
default false maybe ?

Reply:
it can remain true as to simulate the situation before this PR